src/map.js

```diff
@@ -667,12 +669,13 @@ var map = function (arg) {
 
   ////////////////////////////////////////////////////////////////////////////
   /**
-   * Resize map (deprecated)
+   * Resize map in pixels.
```

Comment:
We use this function when the window is resized via the `resizeSelf` function.  We don't do *anything* with x and y (just with width and height).

Since prior to this PR, it has been labelled deprecated for a while, we could remove the function and roll the needed functionality into `resizeSelf`.  We could remove x and y from the `geo.event.resize` event properties, too.